### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by yihfei**
Added new Ui mockup and changed the Readme description to fit our project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+# TuitionConnect
+
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S1-CS2103T-F10-4/tp/actions)
```
Comment (lambraydon) [inline README.md:3]: Nice changes for CI Status

File: README.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+### Description
+TuitionConnect is a powerful **CLI** application designed to streamline the management of your tutoring business for **private tutors**. TuitionConnect helps to effortlessly *manage students*, *schedules* and *progress tracking* while ensuring *financial organization*.
+
+### Credits
+This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
```
Comment (lambraydon) [inline README.md:11]: Good use of GFMD.

**Review by lambraydon — CHANGES_REQUESTED** [review]
Good changes for README.md. However you merged to tutorial-adding-command branch as well which are changes not relevant to this commit. Should we leave out the changes for the additions to RemarkCommand and only include changes to the README?

**Comment by yihfei** [comment]
Closed as code did not pass CI checks

**Pull request closed without merging**